### DISCUSSION TITLE
CASE CLOSED

### DISCUSSION THREAD

**Discussion by panism** [getlantern/lantern-forum-cn#128] [Q&A]

**Comment by MaxMKMax** [comment]
Hi @panism , Sorry for the inconvenience caused. May i know which payment method you are using for the pro plan? There are some issues in automatically upgrading your plan. Please help us to do an in app report within the app/ web and return me the 8 digit reporting tickets by commenting below. The in app report/ log number can be found in your email after you submit the issue report. Appreciate that. Thank you so much.

On my side, i will help you to inform our payment team to manually upgrade your plan to Pro version. Please provide me your email address that you registered for your account. Lantern team will help you to upgrade as soon as possible. Thank you.

**Comment by MaxMKMax** [comment]
@panism Okay great. Let me check with the payment team. I will get back to you tomorrow or in 2 working days. Please follow up with me if your status has not been upgraded to pro plan. Thanks.

**Comment by MaxMKMax** [comment]
@panism Okay noted. Thanks for your information. I already inform the payment team last night and they should be completed proceed to upgrade the plan to pro version for you. May i know what is your status of plan now? Are you able to be successfully upgraded?

**Reply by MaxMKMax** [reply]
Hi @panism , Good evening. The pro account will bonded with the email you registered for the lantern account and the purchased account. 
One pro account available to connect for 3 devices. 
Your situation is slightly different. Let me check for you and get back to you again. Sorry for waiting. Thanks!

**Comment by MaxMKMax** [comment]
@panism May i know which email do you sent for the payment proof? support@getlantern.org ?
Would you mind to let me know which email address you use to send the email to our customer support? Let me ask the payment team to trace it and get your account recovered with the pro plan. Thanks.
